proto/skaffold.proto

```diff
@@ -560,6 +562,10 @@ enum SuggestionCode {
     // Check if docker is running
     CHECK_DOCKER_RUNNING = 106;
 
+    // Deploy error suggestion codes
+
+    // Check cluster connection
+    CHECK_CLUSTER_CONNECTION = 201;
```

Comment:
Please add a new suggestion code  `CHECK_MINIKUBE_STAUTUS` and use that if cluster is minikube